src/wgpu_engine.rs

```diff
@@ -25,13 +25,45 @@ pub struct WgpuEngine {
     pool: ResourcePool,
     bind_map: BindMap,
     downloads: HashMap<Id, Buffer>,
+    pub(crate) use_cpu: bool,
 }
 
-struct Shader {
+struct WgpuShader {
     pipeline: ComputePipeline,
     bind_group_layout: BindGroupLayout,
+}
+
+pub enum CpuShaderType {
+    Present(fn(u32, &[CpuBinding])),
+    Missing,
+    Ununsed,
```

Comment:
Was this meant to be `Unused`?

Reply:
Please fix this typo (this should be `Unused` not `Ununsed`). Also, following the thread above, I think I prefer to call this `Skipped`.